pages/linux/vgscan.md

```diff
@@ -0,0 +1,13 @@
+# vgscan
+
+> Scan for volume groups on all supported Logical Volume Manager (LVM) block devices.
+> See also `lvm`, `vgchange`.
+> More information: <https://manned.org/vgscan>.
+
+- Scan for volume groups and print information about each group found:
+
+`sudo vgscan`
```

Comment:
```suggestion
`vgscan`
```

Reply:
should sudo be used, even if its needed by the command

Reply:
@CleanMachine1, I was following point 2 in the guidelines:

`
When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. For example, commands that require sudo should include it directly in the examples.
`

Reply:
If numbers is what you want @CleanMachine1, try searching 😆
```console
❯ rg '^`sudo ' | wc -l
     849

Reply:
I did actually search some results using VSCode since it has a nice little word search thing in its replace tool

Reply:
I was going to mention it but other things are inconsistent for example

 `tldr dd`

dd needs root to copy from things like block devices.

Reply:
`dd` could and should be updated.